hazelcast-sql/src/test/java/com/hazelcast/sql/SqlErrorAbstractTest.java

```diff
@@ -79,125 +72,9 @@ protected void checkTimeout(boolean useClient, int dataSetSize) {
 
         populate(instance1, dataSetSize);
 
-        // Block batches from instance2 to instance1.
-        BlockingExec.Blocker blocker = new BlockingExec.Blocker();
-
-        setExecHook(instance2, exec -> {
-            if (exec instanceof MapScanExec) {
-                return new BlockingExec(exec, blocker);
-            } else {
-                return exec;
-            }
-        });
-
         // Execute query on the instance1.
-        try {
-            HazelcastSqlException error = assertSqlException(useClient ? client : instance1, query().setTimeoutMillis(100L));
-            assertErrorCode(SqlErrorCode.TIMEOUT, error);
```

Comment:
The method does not seem to be checking anything.

Reply:
To be honest, it's hard to determine that exactly timeout was the reason of failed job. 
If it is not enough, we have timeout tests in light job test suite.